cmd/background-controller/main.go

```diff
@@ -82,14 +83,18 @@ func createrLeaderControllers(
 
 func main() {
 	var (
-		genWorkers      int
-		maxQueuedEvents int
-		omitEvents      string
+		genWorkers                   int
+		maxQueuedEvents              int
+		omitEvents                   string
+		disableAPICallSizeProtection bool
+		maxAPICallResponseLength     int64
 	)
 	flagset := flag.NewFlagSet("updaterequest-controller", flag.ExitOnError)
 	flagset.IntVar(&genWorkers, "genWorkers", 10, "Workers for the background controller.")
 	flagset.IntVar(&maxQueuedEvents, "maxQueuedEvents", 1000, "Maximum events to be queued.")
 	flagset.StringVar(&omitEvents, "omit-events", "", "Set this flag to a comma sperated list of PolicyViolation, PolicyApplied, PolicyError, PolicySkipped to disable events, e.g. --omit-events=PolicyApplied,PolicyViolation")
+	flagset.BoolVar(&disableAPICallSizeProtection, "disableAPICallSizeProtection", false, "If true [NOT RECOMMENDED], API call max size won't be checked, this exposes the controller to memory exhaustion attack")
+	flagset.Int64Var(&maxAPICallResponseLength, "maxAPICallResponseLength", 10*1000*1000, "Configure the value of maximum allowed GET response size from API Calls")
```

Comment:
Can we combine into a single parameter? 

If `0` we can then skip the check.

Also, 10MB seems a bit excessive to me for a default. maybe set to 2MB?